runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/internal/layouts/metrics/ScalarsLayout.java

```diff
@@ -129,17 +122,30 @@ public Builder mode(
             return this;
         }
 
+        public Builder label(
+            String label)
+        {
+            this.label = label;
+            return this;
+        }
+
         @Override
-        public GaugesLayout build()
+        public ScalarsLayout build()
         {
             final File layoutFile = path.toFile();
+            MappedByteBuffer mappedBuffer;
             if (mode == Mode.CREATE_READ_WRITE)
             {
                 CloseHelper.close(createEmptyFile(layoutFile, capacity));
+                mappedBuffer = mapExistingFile(layoutFile, this.label);
+            }
+            else
+            {
+                assert mode == Mode.READ_ONLY;
+                mappedBuffer = mapExistingFile(layoutFile, READ_ONLY, this.label);
```

Comment:
Suggest using `readonly` instead of `mode` for consistency with other non-metrics layouts.

Reply:
done